vignettes/dataset-status.Rmd

```diff
@@ -0,0 +1,37 @@
+---
+title: "dataset-status"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{dataset-status}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+Dataset status is recorded below. Please note that due to our release schedule datasets may remain non-functional if broken using the CRAN version for some time even if fixed on GitHub. Also note that transcient issues may affect our testing of datasets and so our checks may indicate occassionally show a spurious failure. 
+
+```{r}
+library(covidregionaldata)
+library(dplyr)
+library(knitr)
+datasets <- get_available_datasets() %>%
+  arrange(country) %>%
+  select(Region = country, Method = class) %>%
+  mutate(
+    `GitHub status` = paste0(
+      "![", Method,
+      "](https://github.com/epiforecasts/covidregionaldata/workflows/",
+      Method,
+      "/badge.svg)](https://github.com/epiforecasts/covidregionaldata/actions/ workflows/", # nolint
+      Method, ".yaml"
+    ),
+    `CRAN status` = "*working*"
+  )
```

Comment:
Replace this with

```
  mutate(
    `GitHub status` = paste0(
      "![", Method,
      "](https://github.com/epiforecasts/covidregionaldata/workflows/",
      Method,
      "/badge.svg) [", Method,
      "](https://github.com/epiforecasts/covidregionaldata/actions/workflows/", # nolint
      Method, ".yaml)"
    ),
    `CRAN status` = "*working*"
  )
```

Reply:
Updated - nice spot on  the space in workflows!